.github/workflows/pr-test.yml

```diff
@@ -68,7 +68,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
BTW 3.8 is [not officially supported](https://devguide.python.org/versions/), 
@tsmithv11 should we remove it?

Reply:
Let's take this to DM.

Reply:
Not dropping support yet.